docs/backend-system/building-backends/08-migrating.md

```diff
@@ -210,6 +210,25 @@ argument of `createBackend`.
 
 :::
 
+## Removing `@backstage/backend-common`
+
+The `@backstage/backend-common` package has been deprecated as part of moving to the new backend system, an you will need to replace existing usage of it. All exports from the package have been marked as deprecated in the last few releases of the package, and each export has its own deprecation message that explains how to replace that particular export.
+
+These are the deprecation messages for the most common replacements:
+
+- `createLegacyAuthAdapters` - Migrate to use the new backend system and auth services instead.
+- `errorHandler` - Use `MiddlewareFactory.create.error` from `@backstage/backend-defaults/rootHttpRouter` instead.
+- `getRootLogger` - This function will be removed in the future. If you need to get the root logger in the new system, please check out this documentation: https://backstage.io/docs/backend-system/core-services/logger
+- `getVoidLogger` - This function will be removed in the future. If you need to mock the root logger in the new system, please use `mockServices.logger.mock()` from `@backstage/test-utils` instead.
```

Comment:
```suggestion
- `getVoidLogger` - This function will be removed in the future. If you need to mock the root logger in the new system, please use `mockServices.logger.mock()` from `@backstage/backend-test-utils` instead.
```

Reply:
Ah shit, that's what's in the published deprecation message 😅. Alright, good to have it fixed here at least